support/notulen-queries.js

```diff
@@ -28,7 +28,9 @@ async function getZittingForNotulen(uuid) {
         besluit:behandelt ?agendapunten;
         besluit:isGehoudenDoor ?bestuursorgaanUri;
         besluit:geplandeStart ?geplandeStart;
-        mu:uuid ${sparqlEscapeString(uuid)}.
+        mu:uuid ${sparqlEscapeString(uuid)};
+        ext:intro ?intro;
```

Comment:
both intro and outro are best wrapped in an optional block, as they're not required

Reply:
they get set at zitting creation as empty strings